core/templates/pair.h

```diff
@@ -31,50 +31,30 @@
 #ifndef PAIR_H
 #define PAIR_H
 
-#include "core/templates/hashfuncs.h"
 #include "core/typedefs.h"
+
 template <typename F, typename S>
 struct Pair {
 	F first;
 	S second;
 
-	Pair() :
-			first(),
-			second() {
-	}
+	constexpr Pair() :
```

Comment:
Perhaps this can be `= default`?

Reply:
Somehow, this fixed the _actual_ error, but there's still an _entirely unrelated_ false positive. I'll look into this further to see if that can just be addressed directly, or if I still need the warning suppression after all.

Reply:
I'll leave the warning suppression as it was for now; gonna try an alternative solution in a followup PR that's broader in scope.